spdx/document.py

```diff
@@ -175,7 +175,10 @@ def package(self, value):
 
     @property
     def files(self):
-        return self.package.files
+        if self.packages:
+            return self.package.files
```

Comment:
As `self.package` just returns the first element of the list `self.packages` this is a workaround to get the files of the first package. If there are no packages the list is empty and the function would return `None`.  This will be changed once the PR concerning  unpackaged files is merged but for now this is just the workaround to deal with an empty list of packages.

Reply:
ok, then :)